bundles/core/org.eclipse.smarthome.core.thing.xml.test/src/test/resources/test-bundle-pool/ThingTypesTest.bundle/ESH-INF/thing/thing-types.xml

```diff
@@ -10,6 +10,8 @@
 		<description>The hue Bridge represents the Philips hue bridge.
 		</description>
 
+		<category>Light</category>
```

Comment:
This should rather "Gateway" (or what category name we end up for it), shouldn't it?